src/app/centers/createcenter/createcenter.component.html

```diff
@@ -0,0 +1,124 @@
+                  <mat-datepicker #activationDateDatePicker></mat-datepicker>
+                  <mat-error *ngIf="centerForm.controls.activationDate.hasError('required')">
+                    Activation Date is <strong>required</strong>
+                  </mat-error>
+                </mat-form-field>
+              </td>
+            </tr>
+          </table>
+
+          <mat-form-field>
+            <mat-label>External Id </mat-label>
+            <input matInput type="number" formControlName="externalId">
+          </mat-form-field>
+
+
+          <mat-form-field class="example-full">
+            <mat-label>Submitted On</mat-label>
+            <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="submittedOnDatePicker" formControlName="submittedOnDate"
+              >
+            <mat-datepicker-toggle matSuffix [for]="submittedOnDatePicker"></mat-datepicker-toggle>
+            <mat-datepicker #submittedOnDatePicker></mat-datepicker>
+            <mat-error *ngIf="centerForm.controls.submittedOnDate.hasError('required')">
+              Submitted On is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+          <table>
+            <tr>
+              <td>
+                <mat-form-field class="example-full">
+                  <mat-label>Select and add groups</mat-label>
+                  <mat-select>
+                    <mat-option *ngFor="let groups of groupData" [value]="groups.id" (click)="display(groups)">
+                      {{ groups.name }}
+                    </mat-option>
+                  </mat-select>
+                </mat-form-field>
+              </td>
+              <td [hidden]="value">
+                <mat-list *ngFor="let lst of groupDataDetail ">
+                  <button type="button" mat-mini-fab color="primary" (click)="add(lst)">  <fa-icon icon="plus"></fa-icon></button>
+                  <mat-list-item>Group Details</mat-list-item>
+                  <mat-divider [inset]="true"></mat-divider>
+                  <mat-list-item>Name: &nbsp;&nbsp;&nbsp;{{lst.name}} </mat-list-item>
+                  <mat-divider [inset]="true"></mat-divider>
+                  <mat-list-item>Id: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;{{lst.id}} </mat-list-item>
+                  <mat-divider [inset]="true"></mat-divider>
+                  <mat-list-item>Office: &nbsp;&nbsp;&nbsp;{{lst.officeName}} </mat-list-item>
+                </mat-list>
+              </td>
+            </tr>
+          </table>
+
+          <div *ngFor="let list of choice" class="class-form">
+            <mat-form-field appearance="outline" class="example-full1">
+              <mat-label> {{list.name}} </mat-label>
+              <input matInput placeholder="Placeholder" disabled="disabled">
+            </mat-form-field>
+            <button type="button" mat-mini-fab color="warn" (click)="delete(list)">  <i class="material-icons">clear</i></button>
```

Comment:
Make similar changes here as specified for add button above. Use font awesome icons only.